roaring/src/bitmap/store/array_store/mod.rs

```diff
@@ -30,6 +30,10 @@ impl ArrayStore {
         ArrayStore { vec: Vec::with_capacity(capacity) }
     }
 
+    pub fn capacity(&self) -> usize {
```

Comment:
We'd at least need to document that this is the capacity of containers, not the capacity of values, or maybe have a different name?